README.md

```diff
@@ -107,6 +107,9 @@ See the [offical Upgrade Guide](http://docs.mattermost.com/administration/upgrad
 * Do not modify the Listen Address in Service Settings.
 * Rarely 'app' container fails to start because of "connection refused" to
   database. Workaround: Restart the container.
+* On Docker version 1.11.2 and 1.11.1 you can have some problems with mattermostdocker_db image (see [issue/26](https://github.com/mattermost/mattermost-docker/issues/26), [issue/6](https://github.com/mattermost/mattermost-docker/issues/6).
+
+  Please use docker version [1.9.1](https://yum.dockerproject.org/repo/main/centos/7/Packages/)  if you can. There is apparently no problem with this version and postgres image
```

Comment:
Thanks @yboussard, appreciate your help trying out mattermost-docker, sorry to hear about the compatibility issue. Here is some proposed wording to simplify this known issue for the benefit of users: 

```
[The 3.0.2 release on RHEL 7.2 has compatibility issues with Docker version 1.11 and mattermostdocker_db image](https://github.com/mattermost/mattermost-docker/issues/26). RHEL 7.2 with Docker version [1.9.1](https://yum.dockerproject.org/repo/main/centos/7/Packages/) has been verified to work properly with 3.0.2 release. 
```

Please let me know your thoughts?